sym_api_client_python/clients/message_client.py

```diff
@@ -154,7 +154,25 @@ def list_msg_receipts(self, msg_id):
         url = '/pod/v1/admin/messages/{0}/receipts'.format(msg_id)
         return self.bot_client.example('GET', url)
 
-    def list_stream_attachments(self, stream_id):
+    def list_stream_attachments(self, stream_id: str, since: int, to: int, limit: int = 50, sort_dir: str = "ASC"):
```

Comment:
Params `since` and `to` should be optional as well. Please put a `None` default value. If set to `None`, it should not be sent.